frontend/layouts/default.vue

```diff
@@ -166,7 +171,6 @@ export default {
     }
   },
   mounted () {
-    this.$wikibase.$oauth.autoLoginByCookie()
```

Comment:
why remove this line?

Reply:
its from a older version so...its pushing this change